### DISCUSSION TITLE
3.0 ready to go?

### DISCUSSION THREAD

**Discussion by djmitche** [GothenburgBitFactory/taskwarrior#3270] [General]
According to https://github.com/orgs/GothenburgBitFactory/projects/4 we're just about ready to ship 3.0. There's a documentation PR awaiting review, but that's about it.

However, I don't know how to make a release -- in fact, I likely don't have the necessary permissions.

How should we proceed?
 * What other things do we need to wrap up before shipping?
 * Should we do some pre-releases first?
 * What's the process?

**Comment by holly-hacker** [comment]
I would love an update on this. I just started using TaskWarrior and I want to set up a syncing server, but it seems somewhat wasteful to do that now with 3.0 right around the corner.

**Reply by djmitche** [reply]
Thanks for bumping this! I now have repo admin access, but not answers to the above. Maybe @tbabej or @lauft can chime in? If not, I can "wing it" but that may result in some very visible bumps along the way.

**Comment by gour** [comment]
Hello,
I wonder whether `tasksh` will be still useful after tw's 3.0 release?

**Reply by djmitche** [reply]
I haven't tested it. I expect it will still work. Could you give it a try?
